Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM python:3.8-slim-bullseye
 
+ENV DBT_PROFILES_DIR=/.dbt
+ENV AWS_SHARED_CREDENTIALS_FILE=/.dbt/aws_credentials
```

Comment:
This env var is documented as being supported by both aws cli as well as `boto` (what underlies the athena adapter).

Reply:
This is supported by the Redshift adapter, too, right?